src/index.js

```diff
@@ -38,9 +38,12 @@ function loader(...args) {
 
   // Should the file get cached?
   let cache = true;
-
+// this.fs can be undefined 
+// e.g when using the thread-loader
+// fallback to the fs module
+const stat = this.fs ? this.fs.stat : fs.stat
```

Comment:
Fix indent and `;`

Reply:
The `Apply Suggestion` feature is pretty neat. :)